doc/ref_model/chapters/chapter04.md

```diff
@@ -540,6 +540,47 @@ Whereas:
 <p align="center"><img src="../figures/ch04_one_stop_shop.PNG" alt="one_stop_shop" title="One Stop Shop" width="100%"/></p>
 <p align="center"><b>Figure 4-3:</b> Infrastructure Profiles Catalogue</p>
 
+
+<a name="4.2.7.2"></a>
+#### 4.2.7.2 Backwards Compatibility
+The Reference Model (RM) specification describes a complete infrastructure abstraction including a set of cloud infrastructure hardware and software profiles and compute flavours offered to workloads. The set of defined profiles and flavours will evolve along the releases but at the same time the existing workloads need to be supported.  This means that any CNTT deployed cloud should be backwards compatible and support profiles and flavours from the latest three CNTT releases (N-2, N-1, N) as presented in Figure 4-4. 
+
+
+<p align="center"><img src="../figures/ch04-Backwards-compatibility_overview.png" alt="backwards compatibility" title="backwards compatibility" width="100%"/></p>
+<p align="center"><b>Figure 4-4:</b> Backwards Compatibility</p>
+
+Cloud Infrastructure profiles that are available in CNTT release N deployment can be divided into two categories:
+
+ 1. Cloud infrastructure profiles that are part of CNTT release N. These can be either
+    * new profiles defined in release N or
+    * profiles that were defined in earlier releases but are also supported in release N
```

Comment:
Suggest change language to: "existing profiles from earlier releases that are also considered to be defined in release N"

Reply:
In my understanding these profiles are not redefined in the actual release, but continuosly supported.

Reply:
@CsatariGergely I think we need a way to distinguish profiles that are "supported" due to "backward compatibility" and those that are "supported" because they are also now a part of the new release -- new profiles defined and those from previous releases "promoted" to be considered as part of the new release also. The profiles from previous releases supported for backward compatibility will deprecate earlier then those that are "promoted" to be part of the new release.

Reply:
@pgoyal01 The reason why they are supported does not change the fact, that hey are supported. This section is to explain that the profiles which are supported in a release can be supported due to backward compatibility reasons. 
How about changing the 2nd bullet to "existing profiles from earlier releases that are supported for backward compatibility reasons in release N"?

Reply:
change `new profiles defined in release N or`to `incorporate to become part of new release`

Reply:
Changed now.